server/re/src/irods_re_serialization.cpp

```diff
@@ -969,6 +969,63 @@ namespace irods {
 
             return SUCCESS();
         } // serialize_char_ptr_ptr
+
+
+	static irods::error serialize_openedDataObjInp_ptr(boost::any _p,
+							   serialized_parameter_t& _out) {
+	    try {
+		openedDataObjInp_t *ptr = boost::any_cast<openedDataObjInp_t*>(_p);
+
+		// for a valid ptr, we serialize the immediate properties + keyValPair
+		if (ptr) {
+		    _out["l1descInx"] = std::to_string(ptr->l1descInx);
+		    _out["len"] = std::to_string(ptr->len);
```

Comment:
@trel how do we feel about using all of the letters here?

Reply:
Big fan of using the letters.

But on the other hand, this reflects the existing data structure itself... so, unless we're expanding all the rest of them, I think we just leave it.   We can use more letters when we write wrappers or a new thing.